src/applications/caregivers/containers/IntroductionPage.jsx

```diff
@@ -42,18 +42,24 @@ const IntroductionPage = ({ route, router }) => {
             </p>
 
             <ul className="process-lists">
-              <li>Address</li>
-              <li>Telephone number</li>
-              <li>Date of birth</li>
-              <li>Social Security number or Tax identification number</li>
-            </ul>
-
-            <p>You’ll need:</p>
-            <ul className="process-lists">
+              <li>
+                The address, telephone number, and date of birth for the Veteran
+                and each family caregiver applicant
+              </li>
               <li>The VA medical center where the Veteran will receive care</li>
               <li>
                 Health insurance information for the Primary Family Caregiver
               </li>
+
+              <li>
+                Veteran's Social Security number or tax identification number
+                (This is required for the online application only. If you’d like
+                to apply without providing this information, you can
+                <a className="vads-u-margin-left--0p5" href="#">
+                  [download the paper form (PDF)]
+                </a>
```

Comment:
Is this supposed to point to a form? Also, it needs a download icon and PDF file size.

Reply:
Yup solving this one now

Reply:
Fixed